web/frontend/src/components/slam-2d-render.vue

```diff
@@ -97,6 +188,13 @@ const disposeScene = () => {
   scene.clear();
 };
 
+const updatePose = async (newPose: commonApi.Pose) => {
+  const x = newPose.getX();
+  const z = newPose.getZ();
+  const baseMarker = scene.getObjectByName('BaseMarker') ?? await makeMarker(baseMarkerUrl, 'BaseMarker', 0.02);
+  baseMarker.position.set(x + baseMarkerOffset.x, 0, z + baseMarkerOffset.z); 
```

Comment:
Can you please add `baseMarkerOffset.y` & provide it here. It's fine if the offset is 0. Trying to keep magic numbers. out of function bodies.